packages/open-next/src/build/generateOutput.ts

```diff
@@ -92,6 +92,8 @@ interface OpenNextOutput {
   };
 }
 
+const indexHandler = "index.hander";
```

Comment:
There was a typo here

Reply:
sorry 😓

Reply:
No problem

Reply:
> sorry 😓

Typos happens to everyone. Your PR made the code more DRY and easier to read. Great work ⭐️